examples/fuse_examples/imaging/classification/knight/make_predictions_file.py

```diff
@@ -80,7 +81,9 @@ def make_predictions_file(
         data = pd.read_json(json_filepath)
         split = {"test": list(data.case_id)}
 
-    dataset = KNIGHT.dataset(data_path, cache_path, split, reset_cache=False)
+    dataset = KNIGHT.dataset(data_path, cache_path, split, reset_cache=reset_cache)
+    if type(dataset) == tuple and len(dataset) == 2:
```

Comment:
Can you explain this if?

Reply:
KNIGHT.dataset can return either one (test mode) or two (train-val mode) datasets. previously the make_predictions_file assumed there is one (test mode), so I made it also support the (train-val) mode.